avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1352,6 +1363,10 @@ private static boolean isEnumSchema(Schema schema) {
            schema.name() != null && schema.name().equals(AVRO_TYPE_ENUM);
   }
 
+  private static boolean isInstanceOfAvroEnum(Object value) {
+    return value instanceof GenericData.EnumSymbol;
```

Comment:
We've checked for IndexedRecord to cover both GenericRecord and SpecificRecord (I think only the conversion from Connect -> Avro uses GenericRecord since there isn't any other option). Should we be checking both `GenericData.EnumSymbol` and for `java.lang.Enum`?